bombastic/indexer/src/lib.rs

```diff
@@ -58,6 +59,12 @@ impl Run {
                     let index = block_in_place(|| {
                         IndexStore::new(&self.storage, &self.index, sbom_index, context.metrics.registry())
                     })?;
+
+                    let package_index: Box<dyn WriteIndex<Document = (SBOM, String)>> =
+                        Box::new(packages::Index::new());
+                    let packageinfo_index = block_in_place(|| {
```

Comment:
I think it would be good to align the naming a bit. Maybe go with `*_index` and `*_store`?